### PR TITLE
Fix CI.

### PR DESCRIPTION
**Pull request by proofit404**

### CODE REVIEW

**Comment by sobolevn** [comment]
Please, do not forget to add `py.typed` file.

I would also appreciate some tests with https://github.com/typeddjango/pytest-mypy-plugins

**Comment by proofit404** [comment]
`py.typed` and `pytest-mypy-plugin` will be implemented as part of #46

File: tests/helpers/examples/attrs.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ class Chat:
 
 
 @attrs
-class Message:
+class Message(object):
```
Comment (sobolevn) [inline tests/helpers/examples/attrs.py]: :trollface:

Comment (proofit404) [reply tests/helpers/examples/attrs.py]: Python 2 support stands for maturity of the library.

**Pull request merged**